ecosystem.toml

```
@@ -54,8 +58,78 @@ repo = "https://github.com/jacklund/repl-rs"
 crates = ["repl-rs"]
 topics = ["repl"]
 
+[[project]]
+name = "cranelift"
+description = "Cranelift is a low-level retargetable code generator. It translates a target-independent intermediate representation into executable machine code."
+repo = "https://github.com/bytecodealliance/wasmtime/tree/main/cranelift"
+crates = ["cranelift-codegen", "cranelift-frontend", "cranelift-jit", "cranelift", "cranelift-module", "cranelift-codegen-meta", "cranelift-wasm", "cranelift-native", "cranelift-reader", "cranelift-object"]
+topics = ["runtime"]
+
+[[project]]
+name = "llvm-sys.rs"
+description = "Rust bindings to LLVM's C API."
+repo = "https://gitlab.com/taricorp/llvm-sys.rs"
+crates = ["llvm-sys"]
+topics = ["runtime"]
+
+[[project]]
+name = "Inkwell"
+description = "It's a New Kind of Wrapper for Exposing LLVM (Safely)"
+repo = "https://github.com/TheDan64/inkwell"
+crates = ["inkwell"]
+topics = ["runtime"]
+
+[[project]]
+name = "HVM"
+description = "A massively parallel, optimal functional runtime in Rust"
+repo = "https://github.com/Kindelia/HVM"
+crates = ["HMV"]
+topics = ["runtime"]
+
 [[showcase]]
 name = "AbleScript"
 description = "AbleScript is a procedural, cursed, dynamically-typed, interpreted programming language, designed to be bad."
 repo = "https://git.ablecorp.us/AbleScript/able-script"
 docs = "https://ablecorp.github.io/able-script-the-book/"
+
+[[showcase]]
+name = "Rust"
+description = "Empowering everyone to build reliable and efficient software."
+repo = "https://github.com/rust-lang/rust/"
+docs = "https://doc.rust-lang.org/stable/std/index.html"
+
+[[showcase]]
+name = "Roc"
+description = ""
```

`description` is not a required field.
```suggestion
```

---

```suggestion
description = "A language for making delightful software. "
```
That's the repository description, and we probably want *something*.